wp-content/themes/oceanwp-child/page-initiative.php

```diff
@@ -87,7 +87,7 @@
 									
 								}
 							}?>
-							<p class="ini-supporters"><b>30 Supporters</b></p>
+							<p class="initiative-supporters"><b>30 Supporters</b></p>
 								<a href="<?php the_permalink(); ?>"><h4 style="margin-bottom:10px"><b><?php the_title(); ?></b></h4></a>
```

Comment:
`h4` has inline style and again `<b>` inside of it. See comment regarding typography...